Jenkinsfile

```diff
@@ -0,0 +1,10 @@
+pipeline {
+    agent any 
```

Comment:
set agent to

    agent{
        label 'jenkins-workers'
    }

Reply:
once tested, you must add following blocks
**options** for logs mangement
**environment** to make placeholder available for future variables